### DISCUSSION TITLE
Missing Filter.h file

### DISCUSSION THREAD

**Discussion by danschueler** [evert-arias/EasyButton#76] [Q&A]
When compiling EasyButton for use on an ESP32, I'm getting an error
Filter.h: No such file or directory

Where is this file supposed to be?  I'm using Arduino 2.0.3.

**Comment by arkadiuszkocik** [comment]
Hi, I have the same problem. Did you manage to solve it?
Regards

**Reply by MikeGarber** [reply]
2 ways to go
* create your own filter class (not terribly hard) just for the easyButtonTouch
* comment out or delete the easyButtonTouch files
